packages/magento-wishlist/components/WishlistFab/WishlistFab.tsx

```diff
@@ -60,17 +54,11 @@ export function WishlistFab(props: WishlistFabProps) {
 
   const { loggedIn } = useCustomerSession()
 
-  const { data: GetCustomerWishlistData } = useCustomerQuery(GetIsInWishlistsDocument)
-  const { data: guestWishlistData } = useGuestQuery(GuestWishlistDocument)
+  const wishlistData = useWishlistItems()
 
-  let activeWishlist = false
-  if (loggedIn) {
-    const wishlistItemCount = GetCustomerWishlistData?.customer?.wishlists[0]?.items_count || 0
-    activeWishlist = wishlistItemCount > 0
-  } else {
-    const wishlist = guestWishlistData?.guestWishlist?.items || []
-    activeWishlist = wishlist.length > 0
-  }
+  const wishlist = wishlistData.data ?? []
```

Comment:
`useWishlistItems().data` might need to have an empty array as fallback value since there are a lot of places where you give it an empty array as fallback anyway